src/main/java/chatsy/tasks/EventTask.java

```diff
@@ -0,0 +1,80 @@
+package chatsy.tasks;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents a task that spans a specific time range, such as an event.
+ */
+public class EventTask extends Task {
+    private LocalDateTime from;
+    private LocalDateTime to;
+
+    /**
+     * Constructs an {@code EventTask} with the specified description, start time, and end time.
+     *
+     * @param description The description of the event.
+     * @param from The start time of the event.
+     * @param to The end time of the event.
+     */
+    public EventTask(String description, LocalDateTime from, LocalDateTime to) {
+        super(description);
+        this.from = from;
+        this.to = to;
+    }
+
+    /**
+     * Constructs an {@code EventTask} with the specified description, start time, end time, and completion status.
+     *
+     * @param description The description of the event.
+     * @param from The start time of the event.
+     * @param to The end time of the event.
+     * @param isDone The completion status of the event.
+     */
+    public EventTask(String description, LocalDateTime from, LocalDateTime to, boolean isDone) {
+        super(description);
+        this.from = from;
+        this.to = to;
+        this.isDone = isDone;
+    }
+
+    /**
+     * Returns the start time of this event.
+     *
+     * @return The start time as a {@code LocalDateTime}.
+     */
+    public LocalDateTime getFrom() {
+        return from;
+    }
+
+    /**
+     * Returns the end time of this event.
+     *
+     * @return The end time as a {@code LocalDateTime}.
+     */
+    public LocalDateTime getTo() {
+        return to;
+    }
+
+    /**
+     * Returns a string representation of this event, including its status, start time, and end time.
+     *
+     * @return The string representation of the event.
+     */
+    @Override
+    public String toString() {
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("MMM d yyyy HH:mm");
+        return "[E]" + getStatusIcon() + description + " (from: " + from.format(formatter) + " to: " + to.format(formatter) + ")";
```

Comment:
Please try to use line wrapping at appropriate places of the line, as line length should be no longer than 120 chars.